Assets/Scripts/zombieScript.cs

```diff
@@ -113,6 +115,8 @@ protected override void Attack()
     {
         agent.isStopped = true; // NavMeshAgent.Stop is obsolete. Set NavMeshAgent.isStopped to true.
 
+        Actions.OnEnemyDamaged?.Invoke(targetEnemy, damage);
```

Comment:
same here, remove action and call take damage of target enemy

Reply:
Done